content/en/infrastructure/faq/set-up-orchestrator-explorer-daemonset.md

```diff
@@ -100,11 +100,22 @@ This page contains instructions for setting up the Orchestrator Explorer using a
 3. The Process Agent, which runs in the Agent DaemonSet, must be enabled and running (it doesn't need to run the process collection), and configured with the following options:
 
     ```yaml
-    - name: DD_ORCHESTRATOR_EXPLORER_ENABLED
-      value: "true"
+    containers:
+        - name: process-agent
+          env:
+          - name: DD_ORCHESTRATOR_EXPLORER_ENABLED
+            value: "true"
+    ```
+    For Agent versions 7.51.0+, the orchestrator check runs on the Agent container instead of the Process Agent container so configure the following on the Agent container instead:
```

Comment:
```suggestion
    For Agent versions 7.51.0+, the orchestrator check runs on the Agent container instead of the Process Agent container. To configure the Agent container:
```